ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -1201,6 +1201,8 @@ def bind_ovs_ports(self, br_name, dut_iface, injected_iface, vm_iface, disconnec
                            (br_name, dut_iface_id, vm_iface_id))
             VMTopology.cmd("ovs-ofctl add-flow %s table=0,priority=8,udp6,in_port=%s,udp_src=161,action=output:%s,%s" %
                            (br_name, dut_iface_id, vm_iface_id, injected_iface_id))
+            VMTopology.cmd("ovs-ofctl add-flow %s table=0,priority=6,udp6,in_port=%s,udp_dst=4784,action=output:%s" %
```

Comment:
what is dst udp port 4784? Is it for BFD protocol ?

Reply:
Correct. udp6 and dst port 4787 is for bfd multihop protocol.